build-system/tasks/performance/index.js

```diff
@@ -28,11 +29,12 @@ const {printReport} = require('./print-report');
  */
 async function performance() {
   installPackages(__dirname);
-  const {headless, runs, urls} = new loadConfig();
+  const {headless, runs, urls, handlers} = new loadConfig();
```

Comment:
This is so so confusing. 
We first flat the config object to an array of urls and the raw handlers config. 
Then later in `measureDocument` we iterate the handler config and find the corresponding handler type again? 
Could you please optimize the design a bit. Thanks.